src/main/java/com/oath/halodb/HaloDB.java

```diff
@@ -92,6 +92,14 @@ public void resumeCompaction() {
         dbInternal.resumeCompaction();
     }
 
+    public void mergeTombstoneFiles() throws HaloDBException{
```

Comment:
Since this is a new API people can use, do we need to add it into README? Also we can tell users, this is a blocking API, don't run it on a response time/latency sensitive worker thread :)

Also, add a space would be better: `HaloDBException{` -> `HaloDBException {`

Reply:
Good suggestion.  Will update README by including all changes.